YellowNotes.Api/Startup.cs

```diff
@@ -16,8 +26,58 @@ public Startup(IConfiguration configuration)
         public IConfiguration Configuration { get; }
 
         public void ConfigureServices(IServiceCollection services)
-        {
+        {    
+            var allowedHosts = Configuration.GetSection("CorsSettings:AllowedHosts")
+                .Get<string[]>();
+            var allowedMethods = Configuration.GetSection("CorsSettings:AllowedMethods")
+                .Get<string[]>();
+                
+            services.AddCors(options =>
+            {
+                options.AddDefaultPolicy(
+                builder =>
+                {
+                    builder.WithOrigins(allowedHosts)
+                                        .AllowAnyHeader()
+                                        .WithMethods(allowedMethods);
+                });
+            });
+
             services.AddControllers();
```

Comment:
you need to decide what you are going to use - either `AddControllers()` or `AddMvcCore()`